packages/amplify-appsync-simulator/src/server/index.ts

```diff
@@ -2,27 +2,57 @@ import { OperationServer } from './operations';
 import { SubscriptionServer } from './subscription';
 import { AmplifyAppSyncSimulator } from '..';
 import { AppSyncSimulatorServerConfig } from '../type-definition';
+import { Server, createServer } from 'http';
+import portFinder from 'portfinder';
+import e2p from 'event-to-promise';
+import { address as getLocalIpAddress } from 'ip';
+import { AppSyncSimulatorSubscriptionServer } from './websocket-subscription';
+
+const BASE_PORT = 8900;
+const MAX_PORT = 9999;
 
 export class AppSyncSimulatorServer {
   private operationServer: OperationServer;
   private subscriptionServer: SubscriptionServer;
+  private httpServer: Server;
+  private realTimeSubscriptionServer: AppSyncSimulatorSubscriptionServer;
 
-  constructor(config: AppSyncSimulatorServerConfig, simulatorContext: AmplifyAppSyncSimulator) {
+  private _url: string;
```

Comment:
nit: only this private var has _ prefix, lets use common form until we enforce lint at this level